### PR TITLE
K1 drivers side menu 

### PR DESCRIPTION
**Pull request by Waleed2207**
1. Adding LinearGradient //Replace with your desired gradient colors
2. Adding the profile icon and title at the top of the side menu

### CODE REVIEW

**Comment by Waleed2207** [comment]
finish the side menu

File: package.json [hunk 1]
```diff
@@ -10,17 +10,25 @@
...
     "appcenter-analytics": "^4.4.5",
     "appcenter-crashes": "^4.4.5",
+    "expo-font": "^11.1.1",
     "i18next": "^22.5.1",
     "react": "18.2.0",
     "react-i18next": "^12.3.1",
     "react-native": "0.71.7",
+    "react-native-gesture-handler": "^2.12.0",
+    "react-native-linear-gradient": "^2.7.2",
+    "react-native-paper": "^5.8.0",
```
Comment (henaharon) [inline package.json:27]: Why did you install this library?

File: package.json [hunk 2]
```diff
@@ -10,17 +10,25 @@
...
   },
   "dependencies": {
+    "@expo/vector-icons": "^13.0.0",
+    "@react-navigation/drawer": "^6.6.2",
     "@react-navigation/native": "^6.1.6",
     "@react-navigation/native-stack": "^6.9.12",
     "appcenter": "^4.4.5",
     "appcenter-analytics": "^4.4.5",
     "appcenter-crashes": "^4.4.5",
+    "expo-font": "^11.1.1",
```
Comment (henaharon) [inline package.json:20]: Why did you install this library?

File: package.json [hunk 3]
```diff
@@ -10,17 +10,25 @@
     "test": "jest"
   },
   "dependencies": {
+    "@expo/vector-icons": "^13.0.0",
```
Comment (henaharon) [inline package.json:13]: Why did you install this library?

**Comment by Waleed2207** [comment]
I have corrected all comments

**Comment by Waleed2207** [comment]
**`Overall Description:`**

The provided code defines a custom drawer content component for a navigation drawer in a MyCar application. This component is responsible for rendering the content that appears inside the drawer menu.

The drawer content includes the following elements:

1. User information: This section displays information about the user, such as their profile picture, name, and other details.
2. Notifications: This section shows any notifications or alerts for the user. 
3. Menu items: The menu items are rendered using the CommonCard component, which is reused for each item. The menu items are based on the data in the sideMenuMock array. Each CommonCard component displays a title, an icon, and a count. Clicking on a menu item triggers a navigation action based on the item's index. 
4. Line separator: A horizontal line is rendered to visually separate the menu items from the rest of the content.
5. Contact Us: A text section is displayed with translated text indicating how to contact the application's support or customer service.
6. Additional menu items: More menu items are rendered using the CommonCard component for the remaining items in the sideMenuMock array. These menu items have similar properties to the previous ones.
7. Footer: A footer section is displayed at the bottom of the drawer content.

The code also includes a set of styles defined using the StyleSheet.create function.

In summary, this code sets up the layout and functionality of the custom drawer content component, providing a user-friendly navigation experience for the MyCar application.

**Review by henaharon — APPROVED** [review]
